types/index.d.ts

```diff
@@ -12,6 +12,7 @@ declare namespace mercuriusLogging {
     logBody?: boolean | ((context: MercuriusContext) => boolean)
     logVariables?: boolean
     logRequest?: boolean
+    logMessage?: ((context: MercuriusContext) => string | unknown[])
```

Comment:
Could you update the https://github.com/Eomm/mercurius-logging/blob/main/types/index.test-d.ts file as well?

Reply:
done